front/src/applications/stdcm/components/StdcmResults/StdcmResults.tsx

```diff
@@ -93,6 +98,57 @@ const StdcmResults = ({
     return extractMarkersInfo(outputs.results.simulationPathSteps);
   }, [hasSimulationResults, outputs]);
 
+  const [refSchedule, setRefSchedules] = useState<RefSchedule[]>([]);
```

Comment:
For me, `schedule` refers to train timetables (see `TrainSchedule["schedule"]` in osrdEditoastApi). `Ref` isn't very clear on its own either.

Could you think of a better name?
What you're dealing with here are trains that follow the same itinerary as the train you're considering, so perhaps `similarTrain` would be more appropriate?